src/multiplayer_singlepak_results.c

```diff
@@ -177,7 +177,7 @@ void sub_808207C(void)
                 element->unk18 = 0;
                 element->unk1A = 0x100;
                 element->unk8 = 0;
-#ifndef NONMATCHING
+#ifndef NON_MATCHING
```

Comment:
Nice

Reply:
I was just wondering how you wrote that before, so I searched for both and found this was the only one without the underscore!